sys/cddl/contrib/opensolaris/uts/common/sys/dtrace.h

```diff
@@ -763,9 +769,20 @@ typedef struct dof_actdesc {
 	uint32_t dofa_kind;		/* action kind (DTRACEACT_* constant) */
 	uint32_t dofa_ntuple;		/* number of subsequent tuple actions */
 	uint64_t dofa_arg;		/* kind-specific argument */
-	uint64_t dofa_uarg;		/* user-supplied argument */
+	dtrace_uarg_t dofa_uarg;	/* user-supplied argument */
 } dof_actdesc_t;
 
+#ifdef COMPAT_FREEBSD64
```

Comment:
Sorry, just realized head is a user/kernel header and thus these guards should be:
```suggestion
#if defined(_KERNEL) && defined(COMPAT_FREEBSD64)
```